### PR TITLE
Small updates and app section

### PR DESCRIPTION
**Pull request by llewelld**
Makes some very small changes to the repository roundup and adds in the apps section for the newsletter due 16.06.

### CODE REVIEW

File: 20220610.md [hunk 1]
```diff
@@ -45,3 +45,124 @@ This last fortnight, over the 17 pull requests or activities reported in the ope
...
+
+So we hope you enjoy the app updates below, but while you read them, please remember that these still represent just a small portion of the Sailfish OS apps that are getting love and attention from their authors all the time.
+
+## Octava
+
+Octava from Rustem Abzalov (**arustg**) is a relatively new app, and yet in the last few months has seen multiple updates. It's not a complex app, and doesn't claim to be, billing itself as a "Simple Piano Keyboard". If you want to play simple melodies you might just get away with it, as long as your tune doesn't span multiple octaves, and doesn't contain chords. Octava prodtinkle the ivoriesuces a nice rich piano sound, with some nice animations to highlight the key being played. If you're itching to tinkle the digital ivories, this is your opportunity. The latest update provides some subtle user interface improvements. But let's hope **arustg** adds multitouch support so we can really bash out those choruses! Version 0.3.1 of Octava is available from both the Jolla Store and [OpenRepos](https://openrepos.net/content/arustg/frome).
```
Comment (dcaliste) [inline 20220610.md]: /prodtinkle the ivoriesuces/produces/ maybe ;)

File: 20220610.md [hunk 2]
```diff
@@ -45,3 +45,124 @@ This last fortnight, over the 17 pull requests or activities reported in the ope
...
+
+The obvious competitor to SailBabel is Sebastian Wolf's (**WerkWolf**) Wunderfitz. While Wunderfitz has more features, with its scan-translations, toolbar and in-built dictionary list, there's an elegant simplicity to SailBabel that also makes it feel more responsive and lighter to use.
+
+When it comes to either app, the experience is ultimately tied to the quality of the database in use. I found translating between Finnish and English using the [dict.cc](http://www1.dict.cc) database to be a bit hit-and-miss. Often the stem or *vartalo* word exists is in the database, but because neither app supports fuzzy search, getting at it wasn't always obvious.
```
Comment (dcaliste) [inline 20220610.md]: Maybe myself, but the sentence "Often the stem […] exists is in the database". Maybe a typo with the two verbs, or I'm missing the sentence construction.

File: 20220610.md [hunk 3]
```diff
@@ -45,3 +45,124 @@ This last fortnight, over the 17 pull requests or activities reported in the ope
...
+
+## Frome
+
+Rustem Abzalov (**arustg**) has been prolific in updating apps this fortnight. From is a simple app that makes it easy to calculate percentages, useful if you're calculating interest or tips. The interface is unusual but intuitive, with a the buttons displayed as a row in pairs down the left or &mdash; with a configuration option &mdash; right hand side. A history of calculations then fills up on the edge without the numbers.
```
Comment (dcaliste) [inline 20220610.md]: /From is/Frome is/, missing 'e'.

Comment (dcaliste) [inline 20220610.md]: /with a the buttons/with the buttons/ without the 'a' maybe.

File: 20220610.md [hunk 4]
```diff
@@ -45,3 +45,124 @@ This last fortnight, over the 17 pull requests or activities reported in the ope
...
+
+Pi-hole is particularly useful for use with devices that don't themselves provide ad blocking, including the Sailfish Browser, but also extending to other devices such as smart devices. If you're running the server, you should definitely take a look at Samuel's app. This first release is available from the Jolla Store, [OpenRepos](https://openrepos.net/content/blacksheepdev/porthole) and [Chum](https://build.sailfishos.org/package/show/sailfishos:chum/harbour-porthole).
+
+## Yatzy
+
+We coverd Yatzy recently after it had spent a bit of a multi-year haitus without updates. So it's great to see author **rgrnetalk** giving it some love and providing yet more updates to this rather nice dice game. Yatzy isn't Yahtzee, but it is a related Nordic game that plays along similar lines. Six dice are thrown and a pocker-style categorisation applied to the result. Getting one of each category, with the highest score possible, is the name of the game. It's a careful mix of strategy and luck, with some nice dice animation thrown in for good measure.
```
Comment (dcaliste) [inline 20220610.md]: /coverd/covered/

**Review by dcaliste — CHANGES_REQUESTED** [review]
Some typos I think here and there.

File: 20220610.md (hunk 1 above)
Comment (llewelld) [reply 20220610.md]: Thanks... I must have hit paste at the wrong time! Now fixed.

File: 20220610.md (hunk 3 above)
Comment (llewelld) [reply 20220610.md]: Should now be fixed.

Comment (llewelld) [reply 20220610.md]: Thanks, nice spot. Should now be fixed.

File: 20220610.md (hunk 4 above)
Comment (llewelld) [reply 20220610.md]: Again, another nice spot, thank you.

File: 20220610.md (hunk 2 above)
Comment (llewelld) [reply 20220610.md]: You're right, the sentence was wrong. I've reworked it a bit and hopefully it's clearer now.

**Comment by llewelld** [comment]
Thanks for the thorough proofreading, and also for your excellent original text.

**Comment by dcaliste** [comment]
All good, thanks. There may still be some that both of us didn't spot, but let's keep some excuse for comments on the forum ;-D

**Pull request merged**